src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -58,7 +58,10 @@
             linkedFileData.add(sb.toString());
         }
         if (!linkedFileData.isEmpty()) {
-            files.add(convert(linkedFileData));
+            LinkedFile linkedFile = convert(linkedFileData);
+            if (linkedFile != null) {
+                files.add(linkedFile);
+            }
```

Comment:
We ❤️ Java 8, thus following suggestion

```suggestion
            Optional.ofNullable(convert(linkedFileData)).ifPresent(files::add);
```